src/api/APIAdminResourceDeviceGroup.ts

```diff
@@ -0,0 +1,57 @@
+import APIResource from './APIResource'
+import APIList from './APIList'
+
+
+/**
+ * APIAdminResourceDeviceGroup
+ *
+ * @class
+ * @extends APIResource
+ */
+class APIAdminResourceDeviceGroup extends APIResource {
+
+  /**
+   * /device-groups/{id}
+   *
+   * Constructor
+   */
+  constructor (parent: object, id: number) {
+    if (id == null) {
+      throw new Error('Resource ID cannot be null!');
+    }
+
+    super(parent);
+    this.push('admin', 'device-groups', id);
```

Comment:
this resource does not exist:
https://integration.bitbar.com/api/v2/admin/device-groups
there is:
https://integration.bitbar.com/api/v2/device-groups